Recurly.Tests/PagerTest.cs

```diff
@@ -168,5 +200,74 @@ private Recurly.Client GetEmptyPagerClient()
                 RestClient = mockIRestClient.Object
             };
         }
+
+        private Recurly.Client GetPagerFirstClient()
+        {
+            var page = new Pager<MyResource>()
+            {
+                HasMore = true,
+                Data = new List<MyResource>()
+                {
+                    new MyResource() { MyString = "First Resource" }
+                }
+            };
+            var pageResponse = new Mock<IRestResponse<Pager<MyResource>>>();
+            pageResponse.Setup(_ => _.StatusCode).Returns(System.Net.HttpStatusCode.OK);
+            pageResponse.Setup(_ => _.Headers).Returns(new List<Parameter> { });
+            pageResponse.Setup(_ => _.Data).Returns(page);
+
+            var mockIRestClient = new Mock<IRestClient>();
+            var myParams = new List<Parameter> {
+                new RestSharp.Parameter("limit", "1", ParameterType.QueryString),
+                new RestSharp.Parameter("a", "1", ParameterType.QueryString),
+            };
+
+            // TODO: Find a better way to handle this specifically and the concept at a whole
```

Comment:
For anyone looking, we are going to follow up fixing the request end of the specs in this library. It does a good job of mocking the responses but a bad job of verifying the requests. In the follow up we will address the problem as a whole.